src/Recepies/SearchBar/reducer.js

```diff
@@ -0,0 +1,41 @@
+import {
+  FETCHING_RECIPE_FAIL,
+  FETCHING_RECIPE_START,
+  FETCHING_RECIPE_SUCCESS
+} from './Constants';
+
+export const initialState = {
+  isLoading: false,
+  isFetchingRecipes: false,
+  recipes: [],
+  error: ''
+};
+const MealRecipesReducer = (state = initialState, action) => {
+  switch (action.type) {
+  case FETCHING_RECIPE_START:
+    return {
+      ...state,
+      isLoading: true,
+      isFetchingRecipes: true,
+    };
+  case FETCHING_RECIPE_SUCCESS:
+    return {
+      ...state,
+      isLoading: false,
+      isFetchingRecipes: false,
+      recipes: action.payload,
```

Comment:
I have refactored and changed recipes to now a null value. This would improve code down the components where I would not need this statement `recipes && recipes.length`